File: week09/week09-3.py
#week09-3.py 學習計畫 Math 第2題
#Leetcode 1491. Average Salary Excluding the Minimum and Maximum Salary
class Solution:
    def average(self, salary: List[int]) -> float:
        salary.sort()

        total=0
        for i in range(1,len(salary)-1):
            total += salary[i]
        return total / (len(salary)-2)

File: week09/week09-3.cpp
// week09-3.cpp 學習計畫 Math 第2題
// Leetcode 1491. Average Salary Excluding the Minimum and Maximum Salary
class Solution {
public:
    double average(vector<int>& salary) {
        sort(salary.begin(), salary.end());
        double total = 0;
        for(int i=1;i<salary.size()-1;i++){
            total += salary[i];
        }
        return total / (salary.size()-2);
    }
};